openthread_border_router/rootfs/etc/s6-overlay/scripts/otbr-agent-configure.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/with-contenv bashio
+# ==============================================================================
+# Configure OTBR depending on add-on settings
+# ==============================================================================
+
+if bashio::config.true 'nat64'; then
+    bashio::log.info "Enabling NAT64."
+    ot-ctl nat64 enable
```

Comment:
long term we maybe want to finetune firewall rules to determine which device can do outbound communications ?

Reply:
Uh sure, why not :sweat_smile: 

But I don't think this fits into the add-on configuration system. It probably should go through the REST API. Ideally we'd add support for such a feature in upstream OTBR.